es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,113 @@
+                    :href="item.link">
+                    <div
+                        class="bg-white d-flex justify-content-center mb-50 rounded-circle"
+                        align-v="center">
+                        <div
+                            class="m-100 text-orange">
+                            <component
+                                :is="item.icon"
+                                height="46px"
+                                width="46px" />
+                        </div>
+                    </div>
+                    <p class="'font-weight-bolder mb-200 mb-md-0'">{{ item.label }}</p>
+                </a>
+            </b-col>
+        </b-row>
+    </b-container>
+</template>
+
+<script lang="js">
+import IconCommunitySolar from '../lib-icons/icon-community-solar.vue';
+import IconContactUs from '../lib-icons/icon-contact-us.vue';
+import IconMarketplace from '../lib-icons/icon-marketplace.vue';
+import IconInfoBlogPost from '../lib-icons/icon-info-blog-post.vue';
+
+export default {
+    name: 'EsErrorPage',
+    components: {
+        IconCommunitySolar,
+        IconContactUs,
+        IconMarketplace,
+        IconInfoBlogPost,
+    },
+    props: {
+        errorType: {
+            type: String,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            errorCode: {
+                // eslint-disable-next-line max-len, quote-props
+                '403': "Oops! If you're seeing this message, there's a good chance you have cookies or referrers turned off in your browser.",
+                // eslint-disable-next-line quote-props
+                '404': 'Oops! We are terribly sorry, but there is nothing bright to see here.',
+                // eslint-disable-next-line quote-props
+                '500': 'Oops! We are terribly sorry, but our server is not so bright today. Please try again.',
+                // eslint-disable-next-line quote-props
+                '503': 'Service unavailable: The page you are requesting is temporarily unavailable.',
+            },
+            genericMsg: 'Oops! Something went wrong!',
+            linkItems: [
+                {
+                    icon: 'IconContactUs',
+                    link: 'mailto:hello@energysage.com',
+                    label: 'Contact Us',
+                },
+                {
+                    icon: 'IconCommunitySolar',
+                    link: 'https://www.energysage.com/solar/',
+                    label: 'Learn About Solar',
+                },
+                {
+                    icon: 'IconInfoBlogPost',
+                    link: 'https://news.energysage.com/',
+                    label: 'Informative Blog Post',
+                },
+                {
+                    icon: 'IconMarketplace',
+                    link: 'https://www.energysage.com/market-intro/',
+                    label: 'EnergySage Marketplace',
+                },
+
+            ],
```

Comment:
For these link values can you follow a [similar](https://github.com/EnergySage/es-ds/blob/6ee826c3f953034bbb26f36a48952cac5faa5039/es-vue-base/src/lib-utils/footer-content.js#L6) pattern to how our [footer](https://github.com/EnergySage/es-ds/blob/c26ec0da52be9c05fafafe2213564f55d7768c86/es-vue-base/src/lib-components/EsFooter.vue#L126) is doing it?

We want to be able to control the links per environment and make maintaining them easier.